config/locales/en.yml

```diff
@@ -2736,10 +2736,16 @@ en:
     view: View
   timeline_events:
     show:
-      accepted_on: Reviewed and accepted on %{date}.
-      question_with_number: "Question #%{number}"
-      rejected_on: Reviewed and rejected on %{date}.
-      submitted_on: Submitted on %{date}.
+      submission_from: Submission from
+      submitted_on: Submitted on
+      reviewed_on: Reviewed on
+      status: Status
+      rejected: Rejected
+      accepted: Accepted
+      pending: Pending
+      feedbacks: Feedback(s)
```

Comment:
Use the pluralization feature

Reply:
How should I use that with translation? 
Should I keep key-value pair for both singular as well as plural?

Reply:
Look into the examples in the app 😅. 

You can check `CourseCertificates__Root.issued_count`